draft-ietf-quic-invariants.md

```diff
@@ -178,16 +192,17 @@ version-specific semantics are marked with an X.
 
 A QUIC packet with a short header has the high bit of the first octet set to 0.
 
-A QUIC packet with a short header includes an optional connection ID and no
-version field.  The second bit of that octet (that is, 0x40) determines whether
-the connection ID is present.  If the second bit is cleared, a 64-bit connection
-ID immediately follows the first octet.  If the second bit is set, the remainder
-of the packet has version-specific semantics.
+A QUIC packet with a short header includes a Destination Connection ID.  The
+short header does not include the Connection ID Lengths, Source Connection ID,
+or Version fields.
```

Comment:
Two sentences like 

"A load balancer may uses variable length connection IDs for different clients in order to grease the length. Even though the short header does not have an explicit length field for the connection ID, a load balancer could encode the length of the connection ID in the connection ID with it's own encoding scheme"

might be useful to clarify the intention of not having length fields in the short headers

Reply:
This seems potentially helpful to me.

Reply:
or in the manageability statement...?

Reply:
I think that text on how to construct a connection ID that has the sorts of properties we care about (unlinkability, etc...) would be good things to add to that sort of draft.